src/context/analysisResultStateSlice.ts

```diff
@@ -125,8 +126,7 @@ function generateTableFromApiData(
       feature => feature.properties?.[adminLevelName] === row[adminLevelName],
     );
 
-    const name: string =
-      featureBoundary?.properties?.[adminLevelName] || 'No Name';
+    const name: string = getLocationName(adminLayer, featureBoundary);
```

Comment:
specifying the type here seems uneccessary (from the previous code but we could remove it now)
```suggestion
    const name = getLocationName(adminLayer, featureBoundary);
```